client/components/LearnerViewDirective.js

```diff
@@ -629,10 +629,15 @@ tie.directive('learnerView', [function() {
             feedbackDiv.scrollTop = feedbackDiv.scrollHeight +
               additionalHeightForLoadingIndicator;
             $timeout(function() {
+              // Tasks from the first to current.
+              var orderedTasks = [];
```

Comment:
You might want to look at Array.prototype.slice(): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice?v=control